distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -12,6 +12,7 @@ ALERT;Yamaha Receiver Binding: The configuration parameter names now use lower c
 ALERT;Astro Binding: The 'kilometer' and 'miles' channels have been replaced by a new 'distance' channel
 ALERT;Jeelink Binding: The 'currentWatt' and 'maxWatt' channels have been replaced with 'currentPower' and 'maxPower' channels
 ALERT;WeatherUnderground Binding: A bridge has been added on top of the current things, you need to add a bridge containing your api-key.
+ALERT;ZWave Binding: Major changes have been merged to support features such as security. All things must be deleted and re-added.
```

Comment:
If I understood your post on the forum correctly, should this be "All things (except the controller) must be deleted..."?

Reply:
Maybe we could change it to something like: "To be on the safe side, all things should be deleted and re-added" Or would that be to vague?